Add bare-version without included emojis or shortcuts

ErikMichelson opened the pull request:
This PR introduces an export for a bare-version that does not ship the emoji or shortcut definitions alongside.
You might want to use it, if you're using custom definitions and want to keep the final bundle size as low as possible.

I had to set uglify-js in package.json to version 2, because the newer version 3 dropped support for the "preamble" parameter, resulting in failing build procedures.

Closes #41

puzrin commented:
Comments:

1. don't touch `dist`
2. IMO it worth arrange "index"/"light" via "bare"
3. Some minimal test needed (now no proofs bare.js works at all).

ErikMichelson commented:
Done. :)

 - index and light are arranged via bare
 - dist files remain untouched
 - added test cases for bare version, coveralls reports 100% coverage
 - fixed a bug that an empty emoji definition caused 'undefined' to be inserted after every character

puzrin commented:
LGTM.

Could you join everything into single commit?

ErikMichelson commented:
Done
